pkg/scheduler/actions/enqueue/enqueue.go

```diff
@@ -63,6 +66,15 @@ func (enqueue *enqueueAction) Execute(ssn *framework.Session) {
 			}
 		}
 
+		//Reserve pg min resource in advance.
+		if (job.PodGroup.Status.Phase == scheduling.PodGroupInqueue ||
+			job.PodGroup.Status.Phase == scheduling.PodGroupRunning) &&
+			job.PodGroup.Status.Succeeded != job.PodGroup.Spec.MinMember &&
+			job.PodGroup.Spec.MinResources != nil {
```

Comment:
What if when one pod completed and deleted? FYI: https://github.com/volcano-sh/volcano/issues/457

Reply:
@hzxuzhonghu what's your suggestion here?

Reply:
The pg is running. Why do you add its MinResources?

Reply:
Considering all of the resources that will / already being used for pod group resources.